### PR TITLE
communication guide revision

### PR DESCRIPTION
**Pull request by parispittman**
added a ton of things that were not there, removed some things that aren't relevant, out-of-date processes, etc. 

took this [sheet](https://docs.google.com/spreadsheets/d/1Pmlj5nN_QJ3rRlc-HKOLfRdYY5qCWPMjnNPAdZZ22tg/edit?usp=sharing) and made updates to this doc. 

did i miss anything?

### CODE REVIEW

File: communication/README.md [hunk 1]
```diff
@@ -1,106 +1,193 @@
...
-Some communication has moved into SIG-specific channels - see
-a given SIG subdirectory for details.
+A detailed list of upstream communication platforms and resources for contributors.
+Since upstream contributors are generally consumers, many of our channels intertwine.
+See [Misc Community Resources] for more end user/troubleshooting targeted paths.
+
+## Community Groups
+
+Kubernetes encompasses many projects, organized into [community groups].
+The upstream communication flows through those channels, most
```
Comment (jeefy) [inline communication/README.md]: Nix "The", capitalize U in upstream.

File: communication/README.md [hunk 2]
```diff
@@ -1,106 +1,193 @@
...
+notably in the Special Interest Groups [SIGs] that own the docs and
+codebases. Their communication channels may include mailing lists, slack channels,
+zoom meetings, meeting agenda/notes, and can be found on their READMEs and on the
+community groups/[SIGs] page.
+
+You can actively or passively participate in one of the following ways:
+- the community groups public meeting(s) listed in the above community groups page
+- every Thursday at our [weekly community meeting] over [zoom] at [10am US Pacific Time]
+- intro sessions at KubeCon/CloudNativeCon live or recordings on YouTube[need a link here]
```
Comment (jeefy) [inline communication/README.md]: CNCF Channel link: https://www.youtube.com/channel/UCvqbFHwN-nwalWPjPUKpvTA

File: communication/README.md [hunk 3]
```diff
@@ -1,106 +1,193 @@
...
-If that doesn't answer your questions, or if you think you found a bug,
-please [file an issue].
+### Social Media & Blogs
 
-## Accessing community documents
+**Twitter**  
+- [@kubernetesio]  
+ - owned and operated by CNCF. Contact: social@cncf.io
+- [Last Week in Kubernetes Development]  
+ - owned and operated by Josh Berkus
```
Comment (jeefy) [inline communication/README.md]: I'd link to Josh's twitter or GH, their preference. @jberkus ?

File: communication/README.md (hunk 2 above)
Comment (mrbobbytables) [reply communication/README.md]: A playlist would be ideal, but the CNCF channel link should do for now.

File: communication/README.md [hunk 4]
```diff
@@ -1,106 +1,193 @@
...
+
+## Decisions Are Made Here
+
+The project is very large with a robust community group ecosystem and bubbling up
+information is important. Transparency is necessary and these channels are key:
+
+- [kubernetes-dev] mailing list - all upstream Kubernetes news and discussion.
+Many commmunity groups have charters that state they have to post here for
+certain topics like project wide changes. Joining this mailing list is required
+[k-dev moderators]
```
Comment (mrbobbytables) [inline communication/README.md]: Needs link:
https://git.k8s.io/community/communication/moderators.md#kubernetes-dev

File: communication/README.md [hunk 5]
```diff
@@ -1,106 +1,193 @@
...
+community group meetings, Thursday community call, meet our contributors, office
+hours, and more  
+[YouTube Guidelines]  
 
-To access these documents, you must _subscribe to the kubernetes-dev mailing list_
-as outlined above, as these documents are shared automatically with anyone who
-subscribes to that list. 
+**Kubernetes [Blog]**   
+owned by sig-docs and operated by the
+[blog team]
```
Comment (mrbobbytables) [inline communication/README.md]: Super nitty, but is this intentional to be on a separate line?

**Comment by castrojo** [comment]
/lgtm

I'm ok with merging this despite the nits, we can fix later on. \m/

**Pull request merged**